torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -2052,13 +2072,18 @@ class IrParser {
               // We can only handle output as half, float, and double;
               if (const auto opt_ivalue = toIValue(node->input(3))) {
                 const auto scalar_type = opt_ivalue->toScalarType();
```

Comment:
Maybe worth making a simple helper function since this logic looks repeated frequently.

Reply:
Sounds good. I'll merge those together.